src/types/index.ts

```diff
@@ -1,3 +1,5 @@
+import { RenderHookOptions } from './internal'
```

Comment:
I'd probably move this out of `internal` now

Reply:
I did wonder that, wanted to get your confirmation. i'll move it then merge it